pages/Screenshots.html

```diff
@@ -25,9 +25,9 @@
     <a id="gallery" class="anchor" href="#gallery" aria-hidden="true">
       <span class="octicon octicon-link"></span>
     </a>
-    Gallery
+    Library
   </h3>
-  <p><img src="static/images/gallery.png" alt="Screenshot of gallery page" class="screenshot-image"></p>
+  <p><img src="static/images/library.png" alt="Screenshot of gallery page" class="screenshot-image"></p>
```

Comment:
Alt text should be changed, too.

Are you planning to delete the old image? (Only do so if we don't use it elsewhere.)